diff --git a/backend/copr_backend/msgbus.py b/backend/copr_backend/msgbus.py
index 0c224ef1..d5352041 100644
--- a/backend/copr_backend/msgbus.py
+++ b/backend/copr_backend/msgbus.py
@@ -7,7 +7,13 @@ import logging
 import copy
 import json
 
-from copr_messaging import schema
+try:
+    from copr_messaging import schema
+except ImportError:
+    # fedmsg is optional
+    copr_messaging_enabled = False
+else:
+    copr_messaging_enabled = True
 
 from .constants import BuildStatus
 
@@ -33,6 +39,8 @@ def message_from_worker_job(style, topic, job, who, ip, pid):
     once we can exepect that people are using copr-messaging module (and thus
     we can change message format without affecting the API).
     """
+    if not copr_messaging_enabled:
+        return None
     if style == 'v1':
         content = {
             'user': job.submitter,
